caps/repo.go

```diff
@@ -92,6 +92,12 @@ func (r *Repository) Type(name string) *Type {
 	return nil
 }
 
+// Capability finds and returns the Capability with the given name or nil if it
+// is not found.
+func (r *Repository) Capability(name string) *Capability {
+	return r.caps[name]
```

Comment:
Fixed, had to push with --force because I had some junk staged for an unrelated discussion. Nothing in the "past" above has changed. I've only added the two locking patches below.